src/metabase/lib/field.cljc

```diff
@@ -33,17 +35,9 @@
   [query stage f]
   (f query stage))
 
-(defmethod lib.options/options :field
-  [clause]
-  (last clause))
-
-(defmethod lib.options/with-options :field
-  [[_field id-or-name _opts] options]
-  [:field id-or-name options])
```

Comment:
`options` and `with-options` are no longer multimethods, we will just normalize everything to the new shape now.